client/components/info-tooltip/style.scss

```diff
@@ -2,3 +2,33 @@
 	cursor: help;
 	color: $gray-dark;
 }
+
+&.tooltip.popover.info-tooltip__container {
+	.popover__inner {
```

Comment:
The css for `info-tooltip__container` seems new - did we need new CSS, or did this come from somewhere else?